src/SFA.DAS.AdminService.Web/Controllers/RegisterUserController.cs

```diff
@@ -18,19 +17,19 @@ public class RegisterUserController : Controller
     {
         private readonly IOrganisationsApiClient _organisationsApiClient;
         private readonly IContactsApiClient _contactsApiClient;
-        private readonly IMapper _mapper;
+        private readonly IUserViewModelHelper _userViewModelHelper;
 
-        public RegisterUserController(IContactsApiClient contactsApiClient, IHttpContextAccessor httpContextAccessor, IOrganisationsApiClient organisationsApiClient, IMapper mapper)
+        public RegisterUserController(IContactsApiClient contactsApiClient, IHttpContextAccessor httpContextAccessor, IOrganisationsApiClient organisationsApiClient, IUserViewModelHelper userViewModelHelper)
         {
             _contactsApiClient = contactsApiClient;
             _organisationsApiClient = organisationsApiClient;
-            _mapper = mapper;
+            _userViewModelHelper = userViewModelHelper;
         }
```

Comment:
It's best to leave the private fields and constructor arguments as they were. Changing the "internals" of this controller could have unintended side effects somewhere else. In this case, they don't need to change.

This controller should declare an instance of the `UserViewModelHelper` and pass its private data as constructor parameters in the 3 places where it's used, before calling `GetUserViewModel()`. That's what it's doing in the unit test.

Reply:
Had a chat with Chris about this. We have very similar ideas for the refactor and the difference is mostly terminology. Resolving